st2actions/tests/unit/test_mistral_v2.py

```diff
@@ -146,14 +145,30 @@
 
 
 def process_create(payload):
```

Comment:
Looks like the same set of functions is used in multiple test files.

Maybe we could refactor it in some common mock class - e.g. `MockCUDPublic` which implements those methods...